ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryRoutingFunctionalTest.java

```diff
@@ -313,7 +313,6 @@ public void shouldQueryActiveWhenActiveAliveQueryIssuedToStandby() throws Except
     assertThat(rows_0, hasSize(HEADER + 1));
     KsqlHostInfoEntity host = rows_0.get(1).getSourceHost().get();
     assertThat(host.getHost(), is(clusterFormation.active.getHost().getHost()));
-    assertThat(host.getPort(), is(clusterFormation.active.getHost().getPort()));
```

Comment:
removing this seems to dismiss the point of this test - which is explicitly to check that the active is being queried - right? the fact that the `getHost` lines up is a coincidence because both standby and active are on he same node.

Reply:
perhaps the alternative is to see if an attempt to query the active was made at all? either way, I think we should just disable this test and create a ticket, KQE will triage that work.

Reply:
You and @vvcephei were on the same page - disabling makes more sense in terms of what the test is trying to check. Do we usually remove the full test or just remove the `@Test` label? Either way I'll make a ticket and link this PR to it.

Reply:
Oh, sorry, I sat on my review for too long. I think it'd be better to just delete the test. Disabling it only results in dead code. If the follow-on ticket links to this PR, then the assignee will always be able to resurrect the text of the test if they desire.

Reply:
I'm happy to delete it - but just because you asked usually we add `@Ignore` to disable a test